website_sale_secondary_unit/static/src/scss/website_sale_secondary_unit.scss

```diff
@@ -1,7 +1,14 @@
 .css_secondary_quantity {
     max-width: 150px;
 }
-.oe_website_sale input.js_secondary_quantity {
-    min-width: 48px;
-    text-align: center;
+.oe_website_sale {
+    input.js_secondary_quantity {
+        min-width: 48px;
+        text-align: center;
+    }
+    @include media-breakpoint-down(sm) {
+        .td-psu {
```

Comment:
Do you need this extra class? Isn't there a standard one for this?

Reply:
There's td-img which isn't very generic, I'm afraid.

Reply:
I was thinking on Bootstrap `sm-`, `xs-` classes that are intended for that.

Reply:
Ok, yes. Re-review the new simpler approach.